### PR TITLE
change feature-msbuild versions.

### PR DESCRIPTION
**Pull request by brthor**
@eerhardt @livarcocc @piotrpMSFT

### CODE REVIEW

**Comment by brthor** [comment]
test Ubuntu x64 Release Build please

**Comment by brthor** [comment]
test Ubuntu x64 Release Build

File: branchinfo.txt [hunk 1]
```diff
@@ -4,6 +4,6 @@
 MAJOR_VERSION=1
 MINOR_VERSION=0
 PATCH_VERSION=0
-RELEASE_SUFFIX=feature-msbuild
+RELEASE_SUFFIX=featmsbuild
```
Comment (eerhardt) [inline branchinfo.txt]: Why the shortening?

**Comment by eerhardt** [comment]
Also need to update the new project I just added on Friday:  https://github.com/eerhardt/cli/blob/msbuild-Prototype/src/Microsoft.DotNet.Core.Build.Tasks/project.json

**Comment by eerhardt** [comment]
:shipit: after updating the Build.Tasks assembly.

**Pull request merged**